test/test-x-model.js

```diff
@@ -429,6 +453,24 @@ describe('store', () => {
 });
 
 describe('x-model', () => {
```

Comment:
Note that this test suite (other than that single error test) is completely preserved. Many deprecation warnings are thrown, but the tests still pass.

I tried to make this diff as minimal as possible to make clear that great pains were taken to ensure backwards compatibility 😅